### PR TITLE
Added knockout help icons

### PR DESCRIPTION
**Pull request by nishrana**

### CODE REVIEW

**Comment by smlombardi** [comment]
Note that there was no need for the nucleo-demo spans, as I said yesterday.  Also, note that with HTML5 break tags don't need closing slash, you can do `<br>`.

**Comment by nishrana** [comment]
image

**Comment by nishrana** [comment]
image

**Comment by nishrana** [comment]
Refer the above images. Let me rephrase it for you. It doesn't look correct **without** the nucleo-demo class. Why don't you run my branch and see what I'm talking about.

**Comment by smlombardi** [comment]
I did.  The open circle nucleo icons were twice the size of the others.

When I removed those spans, it looks normal now.

screen shot 2016-04-12 at 10 54 19 am

**Comment by nishrana** [comment]
It looks smaller than the nucleo-mini on design system 

image

**Comment by lombardi-gux** [comment]
they inherit the size of the containing object's font size, wherever someone uses them.  Like font awesome.

On the iconography page I set a larger font size to make it easier for people to see them.


On Apr 12, 2016, at 10:58 AM, Nisha Rana <notifications@github.com<mailto:notifications@github.com>> wrote:


It looks smaller than the nucleo-mini on design system

[image]<https://cloud.githubusercontent.com/assets/9932055/14464604/6ccc032a-009d-11e6-9679-b39c3d105e8e.png>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/PBGUX/designsystem/pull/107#issuecomment-208948730>


________________________________

**Comment by nishrana** [comment]
I see. Victor approved the icon from the nucleo-mini page. But if you're saying that it is the correct size, I'll remove those `<span>`

**Comment by lombardi-gux** [comment]
I did it and merged it in already


On Apr 12, 2016, at 11:14 AM, Nisha Rana <notifications@github.com<mailto:notifications@github.com>> wrote:


I see. Victor approved the icon from the nucleo-mini page. But if you're saying that it is the correct size, I'll remove those <span>

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/PBGUX/designsystem/pull/107#issuecomment-208956695>


________________________________

**Comment by nishrana** [comment]
I guess the whole point of the story was to bring an icon which looks like Horizon. The icon is really tiny and that's not how Horizon has. Let's talk with Victor and make sure we have the right size icon.

**Comment by nishrana** [comment]
image

**Comment by smlombardi** [comment]
All the help icons should be the same size. I know what the story is because I wrote it.  We set a global icon size of 14px.  Also the icon's own css is set to a font-size of inherit.  Let's leave this as is.

**Comment by victorhayes** [comment]
Now that we are using Nucleo, Horizon has to comply to what Nucleo has and use the icons for the size (18px) needed, which are the Mini icons. I don't have a problem with offering the filled and outline versions of the "?" and "i" buttons.

**Comment by victorhayes** [comment]
I do agree with Steve that all the help icons in the content area should be the same.

**Comment by nishrana** [comment]
@victorhayes - It's not 18px. It's 14px as Steve said. 18px would probably have been closer to what Horizon has. I guess I'll let Ania decide what works for Horizon.
